app/components/login_button_component.rb

```diff
@@ -0,0 +1,60 @@
+class LoginButtonComponent < BaseComponent
+  attr_reader :action, :big, :color, :width, :height, :logo_path, :tag_options
+
+  alias_method :big?, :big
+
+  DEFAULT_HEIGHT = 16
+  DEFAULT_WIDTH = 121
+  BIG_HEIGHT = 20
+  BIG_WIDTH = 152
+
+  def initialize(
+    action: ->(**tag_options, &block) { button_tag(**tag_options, &block) },
+    big: false,
+    color: "light blue",
+    **tag_options
+  )
+    @action = action
+    @big = big
+    @color = color
+    @tag_options = tag_options
+  end
+
+  def logo_path
+    return "logo-white.svg" if color == "dark blue"
+    "logo.svg"
+  end
+
+  def width
+    return BIG_WIDTH if big?
+    DEFAULT_WIDTH
+  end
+
+  def height
+    return BIG_HEIGHT if big?
+    DEFAULT_HEIGHT
+  end
+
+  def css_class
+    classes = ['usa-button', *tag_options[:class]]
+    classes << 'usa-button--big' if big
```

Comment:
Rather than applying these classes from this component, could the component render a `ButtonComponent` and rely on the implementation of that component to apply the default classes?

That might also help avoid the duplication of the `content` method below from `ButtonComponent`.